src/js/components/InspectorSidebar.tsx

```diff
@@ -83,43 +94,43 @@ var Inspector = createReactClass({
         primType = TYPES.SCALES;
       }
 
-      InspectorType = Inspector[ctor];
+      // InspectorType = Inspector[ctor]; // TODO: This throws an error when uncommented
```

Comment:
this breaks bc `Inspector` was a var and now it has been renamed to a class named `BaseInspector`

Reply:
you'll want to find out what the possible values of ctor are and adjust accordingly -- probably can't access the properties of BaseInspector using the [] syntax anymore though